static/templates/actors/hazard/sheet-new.html

```diff
@@ -99,14 +99,13 @@ <h3>Actions</h3>
                                     <div class="action-name rollable">
                                         <h4><a>{{action.name}} {{{actionGlyph action.actionCost}}}</a></h4>
                                     </div>
-                                    {{#if ../../owner}}
                                     <div class="item-controls">
-                                        {{#if @root.editing}}
+                                        <a class="item-chat chat" title="{{ localize 'PF2E.NPC.SendToChat'}}"><i class="fas fa-comment-alt"></i></a>
```

Comment:
data-action? :)

Reply:
wouldn't be bad, would also be a change to NPC but I'd be for it.

Reply:
Actually this is more invasive than we think, the sibling elements key using "edit" and "delete" css classes. IMO might be best to leave them all the same for now.